### DISCUSSION TITLE
SOUP to 2.4.20 fails

### DISCUSSION THREAD

**Discussion by nilssachs** [Security-Onion-Solutions/securityonion#11539] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

64

### Storage for /

500GB

### Storage for /nsm

3TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi guys,

the SOUP update to 2.4.20 fails with error:

curl: (35) error:0A000152:SSL routines::unsafe legacy renegotiation disabled
ERROR: Unable to complete so-repo-sync.
Exiting.

Last update worked fine.

Do you have any idea what I could do?

Thanks for your help, Nils

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Acewiza** [comment]
I would probably just run it again and see if the network gremlins found somebody else to annoy.

**Comment by RagnarSTS** [comment]
I hit ctrl-c to stop mine too and attempted a second time.  Second time seemed to halt too but actually didn't.  Just sat for ~30 minutes with zero console change.  Then it suddenly started doing things again.
Feels like server Win2000 again.  
I think it just isn't quite as responsive as we'd all like.

**Comment by defensivedepth** [comment]
As @Acewiza suggested, please try running it again. That error is not specific to Security Onion, but looks like something related to the network. 

@RagnarSTS If you are seeing something similar, please create a new Discussion.

**Comment by TOoSmOotH** [comment, marked as answer]
The issue is there is a bug in soup that does not display the container download portion of soup. If you tail -f the /root/soup.log you will see that is is busy downloading containers in the background. It just isn't being echoed to the screen. This should be fixed when you soup to .30 when .30 is released.

**Reply by nilssachs** [reply]
Thanks! That was the issue. After digging in the logs, I found out that the container downloads did not work properly. After solving some certificate stuff it worked fine.
